returnn/torch/engine.py

```diff
@@ -57,6 +58,12 @@ def __init__(self, config: Config):
 
         self._start_epoch = None  # type: Optional[int]
         self._final_epoch = None  # type: Optional[int]
+        self._min_seq_len = config.typed_value("min_seq_len", None) or config.int(
+            "min_seq_len", 0
+        )  # type: Union[int,float,Dict[str,int],NumbersDict]
+        self._max_seq_len = config.typed_value("max_seq_len", None) or config.int(
+            "max_seq_len", sys.maxsize
+        )  # type: Union[int,float,Dict[str,int],NumbersDict]
```

Comment:
The options are called `min_seq_length` and `max_seq_length`. Why this inconsistency? I would just use the same name.